step_definitions/task-steps.js

```diff
@@ -86,3 +82,21 @@ When(/^.* goes to task evaluation$/, async function () {
 When(/^.* file evaluation is visible$/, async function () {
     await taskPage.checkFileEvaluationStudent(file)
 });
+Then(/^.* see created private task with name '([^']*)'$/, async function (taskName) {
```

Comment:
maybe can see so it makes sense for both singular and plural. See the following lines too.